planetiler-core/src/main/java/com/onthegomap/planetiler/util/CacheByZoom.java

```diff
@@ -29,7 +29,7 @@ private CacheByZoom(int minzoom, int maxzoom, IntFunction<T> supplier) {
    * @return a cache for {@code supplier} by zom
    */
   public static <T> CacheByZoom<T> create(PlanetilerConfig config, IntFunction<T> supplier) {
-    return new CacheByZoom<>(config.minzoom(), config.maxzoom(), supplier);
+    return new CacheByZoom<>(config.minzoomForRendering(), config.maxzoomForRendering(), supplier);
```

Comment:
these zoom levels don't matter too much - they just define the size of the object array that stores cached values, you can just leave with minzoom/maxzoom so that minzoomForRendering isn't necessary.

Reply:
@msbarry actually it fails without this. I end up with an inifinite amount of error about out of bounds (sorry dont have the error here).